### PR TITLE
Abs val neg 2

### PR DESCRIPTION
**Pull request by retabak**
I had some weird merge conflicts (my fault) that I think I have resolved. You only need to review the SECOND absolute value lesson, even though you'll noticed changes to the first. (Those are old.)

I think I've incorporated the feedback that we discussed!

### CODE REVIEW

File: lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/index.adoc [hunk 1]
```diff
@@ -76,15 +79,14 @@ Again, circulate during the activity. As students match Verbal Expressions to th
 -  Fill in the empty spaces so that all three forms represent the same expression.
 }
 
-=== Synthesize
-
-Here are some key take-aways from this lesson:
+Here are some key take-aways:
 
 - We can find the @vocab{opposite} of an @vocab{absolute value}: @math{- | x |}
 - We can find the @vocab{absolute value} of an @vocab{opposite}: @math{ |- x |}
```
Comment (schanzer) [inline lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/index.adoc:85]: Just noticing this now - the spacing on the negative sign is a little weird. Shouldn't it be `|-x|` and `-|x|`?

File: lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/index.adoc [hunk 2]
```diff
@@ -45,29 +42,35 @@ Students match Circles of Evaluation to expressions in words with @vocab{absolut
 
 === Launch
 
-Explain to students that they are going to receive @opt-printable-exercise{pages/translating-card-sort.adoc, three sets of cards}.
+@teacher{During this activity, students sort three separate sets of cards: @handout{translating-card-sort.adoc, "Circles of Evaluation"}, @handout{translating-card-sort-2.adoc, "Verbal Expressions"}, and @handout{translating-card-sort-3.adoc, "Arithmetic Expressions"}. Give each pair of students an envelope containing the three unique sets of cards. Keep each set together with rubber bands or paperclips.}
```
Comment (schanzer) [inline lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/index.adoc:45]: LOVE the way you redid the directions for this activity. Way more readable.

File: lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/index.adoc [hunk 3]
```diff
@@ -93,11 +95,13 @@ And here are some questions to consider:
 
 === Overview
 
-Students tackle evaluating expressions that include both `abs` _and_ `negate`.
+Students evaluate expressions that include both @show{(code 'abs)} _and_ `negate`.
```
Comment (schanzer) [inline lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/index.adoc:98]: Nice simplification!

File: lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/pages/examples.adoc [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+#content table .autonum::after { content: ')'; }
+#content th { text-align: center !important; }
+</style>
+++++
+
+== Predict
+
+@vspace{1ex}
+
+With your partner, predict if each equation on the table below is _true_ when `m` is defined as 6, 12, and 0. Note: an equation might be true for some values and false for others!
```
Comment (schanzer) [inline lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/pages/examples.adoc:16]: You need to tell them what to do with their predictions! Add a sentence about placing a check in the boxes that will pass. (And make sure you do the same for all similar pages)

File: lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/pages/examples.adoc [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+@hspace{2em} Test # @fitb{3em}{@ifsoln{1}} failed because the left side was @fitb{2ex}{@ifsoln{-6}} and the right side was @fitb{2ex}{@ifsoln{6}}.
+
+@hspace{2em} Test # @fitb{3em}{@ifsoln{3}} failed because the left side was @fitb{2ex}{@ifsoln{-6}} and the right side was @fitb{2ex}{@ifsoln{6}}.
+
+@hspace{2em} Test # @fitb{3em}{@ifsoln{5}} failed because the left side was @fitb{2ex}{@ifsoln{6}} and the right side was @fitb{2ex}{@ifsoln{-6}}.
+
+
+@n Talia says that setting `m` equal to _any_ positive value will produce the same results. In other words, she thinks that the *same tests* will fail if `m > 0`? Do you agree? Explain. @fitb{}{@ifsoln{Yes. All positive values will produce the same result.}}
+
+@n Edit the definition of `m` (Section 1) to try out some other *positive* values. Was your prediction correct? Explain. @fitb{}{}
```
Comment (schanzer) [inline lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/pages/examples.adoc:75]: "(Section 1, in the starter file)"

File: lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/pages/examples.adoc [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+@fitb{}{}
+
+@fitb{}{}
+
+@n Change the definition of `m` so that it equals 0. Click "Run". Which tests failed? @fitb{}{@ifsoln{None of them!}}
+
+== Reflect
+
+@n Which equations in the @proglang file are *always* true? @fitb{}{@ifsoln{None of them!}}
```
Comment (schanzer) [inline lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/pages/examples.adoc:93]: Why bold the word "always"? Should you bold "true" and "false"?

File: lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/index.adoc [hunk 7]
```diff
@@ -136,3 +142,52 @@ Students who  can toggle between verbal expressions, Circles of Evaluation, and
...
+
+@lesson-instruction{
+- Complete the final section of @printable-exercise{examples.adoc}.
+- How can you use @proglang to help you decide if an equation is true?
+}
+
+Two algebraic expressions are @vocab{equivalent} if they produce the same outcome, no matter what value we substitute in for the variable(s). In this activity, *none* of the equations that tested *always* passed... which means that none of them were true.
+
+@QandA{
+Can you write an algebraic equation that includes both @show{(code 'abs)} and `negate` which is *always* true?
```
Comment (schanzer) [inline lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/index.adoc:187]: Conventional algebra doesn't have `negate` as a function. I'm guessing you mean "can you write an example/equation in Pyret.."?

**Review by schanzer — CHANGES_REQUESTED** [review]
This is awesome! I made some small comments, but this is very close to merging

File: lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/pages/examples.adoc (hunk 4 above)
Comment (retabak) [reply lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/pages/examples.adoc:16]: struggled with wording here. Feel free to change if it's no good!

File: lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/pages/examples.adoc (hunk 6 above)
Comment (retabak) [reply lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/pages/examples.adoc:93]: yes!

**Comment by retabak** [comment]
@schanzer - done incorporating feedback and also finalized slides. Take a look and LMK if this is ready to merge?

**Comment by schanzer** [comment]
@retabak looks great! I've updated the directions on that one page. If you're happy squash+merge away!

**Comment by retabak** [comment]
yay!

**Pull request merged**